ply not installed.

With python 3 I created a new virtual environment and went through make develop build but ply was not installed for some reason.  Tried this whole process several times and same result.  I did pip install ply and it all worked.

ply is in requirements.txt so that is not the issue but am I missing something???

Here is my trail for:

    mkvirtualenv
    make develop
    make build

==============================

```



mkvirtualenv -a $projenv --python=`which python3` $1
(pywbem27) kschopmeyer@sheldon:~/pywbem/githubpywbem/pywbem(ks/#1187-simple-schema-install)$ mkvirtualenv -a . --python=`which python3` pywbem3
Running virtualenv with interpreter /usr/bin/python3
Using base prefix '/usr'
New python executable in /home/kschopmeyer/.virtualenvs/pywbem3/bin/python3
Also creating executable in /home/kschopmeyer/.virtualenvs/pywbem3/bin/python
Installing setuptools, pip, wheel...done.
virtualenvwrapper.user_scripts creating /home/kschopmeyer/.virtualenvs/pywbem3/bin/predeactivate
virtualenvwrapper.user_scripts creating /home/kschopmeyer/.virtualenvs/pywbem3/bin/postdeactivate
virtualenvwrapper.user_scripts creating /home/kschopmeyer/.virtualenvs/pywbem3/bin/preactivate
virtualenvwrapper.user_scripts creating /home/kschopmeyer/.virtualenvs/pywbem3/bin/postactivate
virtualenvwrapper.user_scripts creating /home/kschopmeyer/.virtualenvs/pywbem3/bin/get_env_details
Setting project for pywbem3 to /home/kschopmeyer/pywbem/githubpywbem/pywbem
(pywbem3) kschopmeyer@sheldon:~/pywbem/githubpywbem/pywbem(ks/#1187-simple-sc

make develop

...  Did not capture the first few entries.

Get:3 http://us.archive.ubuntu.com trusty-updates/restricted Sources [6,322 B]
Hit http://extras.ubuntu.com trusty/main amd64 Packages
Get:4 http://us.archive.ubuntu.com trusty-updates/universe Sources [199 kB]
Hit http://archive.canonical.com trusty Release
Get:5 http://us.archive.ubuntu.com trusty-updates/multiverse Sources [7,373 B]
Hit http://extras.ubuntu.com trusty/main i386 Packages
Get:6 http://us.archive.ubuntu.com trusty-updates/main amd64 Packages [1,072 kB]
Hit http://archive.canonical.com trusty/partner amd64 Packages
Hit http://ppa.launchpad.net trusty InRelease
Hit http://archive.canonical.com trusty/partner i386 Packages
Hit http://archive.canonical.com trusty/partner Translation-en
Hit http://ppa.launchpad.net trusty/main amd64 Packages
Get:7 http://us.archive.ubuntu.com trusty-updates/restricted amd64 Packages [17.2 kB]
Get:8 http://us.archive.ubuntu.com trusty-updates/universe amd64 Packages [450 kB]
Hit http://archive.canonical.com trusty/partner Sources
Hit https://packagecloud.io jessie InRelease
Hit http://archive.canonical.com trusty/partner amd64 Packages
Hit http://ppa.launchpad.net trusty/main i386 Packages
Get:9 http://us.archive.ubuntu.com trusty-updates/multiverse amd64 Packages [14.6 kB]
Hit https://packagecloud.io jessie/main amd64 Packages
Get:10 http://us.archive.ubuntu.com trusty-updates/main i386 Packages [1,013 kB]
Hit http://archive.canonical.com trusty/partner i386 Packages
Hit https://packagecloud.io jessie/main i386 Packages
Hit http://archive.canonical.com trusty/partner Translation-en
Hit http://ppa.launchpad.net trusty/main Translation-en
Get:11 https://packagecloud.io jessie/main Translation-en_US
Get:12 http://us.archive.ubuntu.com trusty-updates/restricted i386 Packages [17.1 kB]
Get:13 http://us.archive.ubuntu.com trusty-updates/universe i386 Packages [444 kB]
Hit http://ppa.launchpad.net trusty/main amd64 Packages
Get:14 http://us.archive.ubuntu.com trusty-updates/multiverse i386 Packages [15.0 kB]
Hit http://us.archive.ubuntu.com trusty-updates/main Translation-en
Ign http://extras.ubuntu.com trusty/main Translation-en_US
Hit http://ppa.launchpad.net trusty/main i386 Packages
Ign http://extras.ubuntu.com trusty/main Translation-en
Ign https://packagecloud.io jessie/main Translation-en_US
Hit http://us.archive.ubuntu.com trusty-updates/multiverse Translation-en
Ign https://packagecloud.io jessie/main Translation-en
Hit http://us.archive.ubuntu.com trusty-updates/restricted Translation-en
Hit http://us.archive.ubuntu.com trusty-updates/universe Translation-en
Hit http://us.archive.ubuntu.com trusty-backports/main Sources
Hit http://ppa.launchpad.net trusty/main Translation-en
Hit http://us.archive.ubuntu.com trusty-backports/restricted Sources
Hit http://us.archive.ubuntu.com trusty-backports/universe Sources
Hit http://us.archive.ubuntu.com trusty-backports/multiverse Sources
Hit http://ppa.launchpad.net trusty Release.gpg
Hit http://us.archive.ubuntu.com trusty-backports/main amd64 Packages
Hit http://us.archive.ubuntu.com trusty-backports/restricted amd64 Packages
Hit http://us.archive.ubuntu.com trusty-backports/universe amd64 Packages
Hit http://us.archive.ubuntu.com trusty-backports/multiverse amd64 Packages
Hit http://ppa.launchpad.net trusty/main amd64 Packages
Hit http://us.archive.ubuntu.com trusty-backports/main i386 Packages
Hit http://us.archive.ubuntu.com trusty-backports/restricted i386 Packages
Get:15 http://security.ubuntu.com trusty-security InRelease [65.9 kB]
Hit http://us.archive.ubuntu.com trusty-backports/universe i386 Packages
Hit http://us.archive.ubuntu.com trusty-backports/multiverse i386 Packages
Hit http://ppa.launchpad.net trusty/main i386 Packages
Hit http://us.archive.ubuntu.com trusty-backports/main Translation-en
Hit http://us.archive.ubuntu.com trusty-backports/multiverse Translation-en
Hit http://us.archive.ubuntu.com trusty-backports/restricted Translation-en
Hit http://ppa.launchpad.net trusty/main Translation-en
Hit http://us.archive.ubuntu.com trusty-backports/universe Translation-en
Get:16 http://security.ubuntu.com trusty-security/main Sources [154 kB]
Hit http://us.archive.ubuntu.com trusty Release
Hit http://us.archive.ubuntu.com trusty/main Sources
Hit http://ppa.launchpad.net trusty Release
Hit http://us.archive.ubuntu.com trusty/restricted Sources
Hit http://us.archive.ubuntu.com trusty/universe Sources
Hit http://us.archive.ubuntu.com trusty/multiverse Sources
Hit http://us.archive.ubuntu.com trusty/main amd64 Packages
Hit http://ppa.launchpad.net trusty/main amd64 Packages
Hit http://us.archive.ubuntu.com trusty/restricted amd64 Packages
Hit http://us.archive.ubuntu.com trusty/universe amd64 Packages
Hit http://us.archive.ubuntu.com trusty/multiverse amd64 Packages
Hit http://us.archive.ubuntu.com trusty/main i386 Packages
Hit http://ppa.launchpad.net trusty/main i386 Packages
Hit http://us.archive.ubuntu.com trusty/restricted i386 Packages
Hit http://us.archive.ubuntu.com trusty/universe i386 Packages
Hit http://us.archive.ubuntu.com trusty/multiverse i386 Packages
Hit http://ppa.launchpad.net trusty/main Translation-en
Hit http://us.archive.ubuntu.com trusty/main Translation-en
Hit http://us.archive.ubuntu.com trusty/multiverse Translation-en
Hit http://us.archive.ubuntu.com trusty/restricted Translation-en
Hit http://us.archive.ubuntu.com trusty/universe Translation-en
Get:17 http://security.ubuntu.com trusty-security/restricted Sources [4,931 B]
Get:18 http://security.ubuntu.com trusty-security/universe Sources [71.9 kB]
Ign http://us.archive.ubuntu.com trusty/main Translation-en_US
Ign http://us.archive.ubuntu.com trusty/multiverse Translation-en_US
Ign http://us.archive.ubuntu.com trusty/restricted Translation-en_US
Ign http://us.archive.ubuntu.com trusty/universe Translation-en_US
Get:19 http://security.ubuntu.com trusty-security/multiverse Sources [3,193 B]
Get:20 http://security.ubuntu.com trusty-security/main amd64 Packages [728 kB]
Get:21 http://security.ubuntu.com trusty-security/restricted amd64 Packages [14.1 kB]
Get:22 http://security.ubuntu.com trusty-security/universe amd64 Packages [225 kB]
Get:23 http://security.ubuntu.com trusty-security/multiverse amd64 Packages [4,800 B]
Get:24 http://security.ubuntu.com trusty-security/main i386 Packages [676 kB]
Get:25 http://security.ubuntu.com trusty-security/restricted i386 Packages [13.9 kB]
Get:26 http://security.ubuntu.com trusty-security/universe i386 Packages [218 kB]
Get:27 http://security.ubuntu.com trusty-security/multiverse i386 Packages [4,955 B]
Hit http://security.ubuntu.com trusty-security/main Translation-en
Hit http://security.ubuntu.com trusty-security/multiverse Translation-en
Hit http://security.ubuntu.com trusty-security/restricted Translation-en
Hit http://security.ubuntu.com trusty-security/universe Translation-en
Fetched 5,921 kB in 9s (648 kB/s)
Reading package lists...
pywbem_os_setup.sh: Installing package: libxml2-utils
Reading package lists... Done
Building dependency tree       
Reading state information... Done
libxml2-utils is already the newest version.
0 upgraded, 0 newly installed, 0 to remove and 60 not upgraded.
touch develop_os.done
makefile: Done installing OS-level development requirements
makefile: Installing/upgrading basic Python packages (pip, etc., with PACKAGE_LEVEL=latest)
python remove_duplicate_setuptools.py
Removing duplicate metadata directories of package: setuptools
Current Python version: 3.4
Version of imported setuptools package: 39.0.1
Site packages directory of imported package: /home/kschopmeyer/.virtualenvs/pywbem3/lib/python3.4/site-packages
Found matching metadata directory: /home/kschopmeyer/.virtualenvs/pywbem3/lib/python3.4/site-packages/setuptools-39.0.1.dist-info
python -m pip install --upgrade pip setuptools wheel
Requirement already up-to-date: pip in /home/kschopmeyer/.virtualenvs/pywbem3/lib/python3.4/site-packages (10.0.1)
Requirement already up-to-date: setuptools in /home/kschopmeyer/.virtualenvs/pywbem3/lib/python3.4/site-packages (39.0.1)
Requirement already up-to-date: wheel in /home/kschopmeyer/.virtualenvs/pywbem3/lib/python3.4/site-packages (0.31.0)
pywbem 0.12.1.dev16 requires mock>=2.0.0, which is not installed.
pywbem 0.12.1.dev16 requires pbr>=1.10.0, which is not installed.
pywbem 0.12.1.dev16 requires ply>=3.10, which is not installed.
pywbem 0.12.1.dev16 requires PyYAML>=3.12, which is not installed.
pywbem 0.12.1.dev16 requires six>=1.10.0, which is not installed.
pip install --upgrade pbr
Collecting pbr
  Using cached https://files.pythonhosted.org/packages/e1/ba/f95e3ec83f93919b1437028e989cf3fa5ff4f5cae4a1f62255f71deddb5b/pbr-4.0.2-py2.py3-none-any.whl
Installing collected packages: pbr
Successfully installed pbr-4.0.2
makefile: Done installing/upgrading basic Python packages
makefile: Installing Python development requirements (with PACKAGE_LEVEL=latest)
pip install --upgrade -r dev-requirements.txt
Ignoring pytest: markers 'python_version == "2.6"' don't match your environment
Ignoring python-coveralls: markers 'python_version == "2.7"' don't match your environment
Ignoring pylint: markers 'python_version == "2.7"' don't match your environment
Ignoring flake8: markers 'python_version == "2.6"' don't match your environment
Ignoring tornado: markers 'python_version <= "2.7"' don't match your environment
Collecting unittest2>=1.1.0 (from -r dev-requirements.txt (line 13))
  Using cached https://files.pythonhosted.org/packages/72/20/7f0f433060a962200b7272b8c12ba90ef5b903e218174301d0abfd523813/unittest2-1.1.0-py2.py3-none-any.whl
Collecting pytest>=3.0.7 (from -r dev-requirements.txt (line 15))
  Using cached https://files.pythonhosted.org/packages/ed/96/271c93f75212c06e2a7ec3e2fa8a9c90acee0a4838dc05bf379ea09aae31/pytest-3.5.0-py2.py3-none-any.whl
Collecting testfixtures<6.0.0,>=4.3.3 (from -r dev-requirements.txt (line 17))
  Using cached https://files.pythonhosted.org/packages/a7/0b/ba062482ebfba44fecd0351cd1017e9c768446bfe9d68fbcd82e545b061f/testfixtures-5.4.0-py2.py3-none-any.whl
Collecting httpretty>=0.8.14 (from -r dev-requirements.txt (line 18))
Collecting lxml>=4.0.0 (from -r dev-requirements.txt (line 19))
  Using cached https://files.pythonhosted.org/packages/9c/8b/92d78357bcbb03e300e45f33a48a322ef458be009b4ed6870a05664128b0/lxml-4.2.1-cp34-cp34m-manylinux1_x86_64.whl
Collecting requests>=2.12.4 (from -r dev-requirements.txt (line 20))
  Using cached https://files.pythonhosted.org/packages/49/df/50aa1999ab9bde74656c2919d9c0c085fd2b3775fd3eca826012bef76d8c/requests-2.18.4-py2.py3-none-any.whl
Collecting decorator>=4.0.11 (from -r dev-requirements.txt (line 21))
  Using cached https://files.pythonhosted.org/packages/bc/bb/a24838832ba35baf52f32ab1a49b906b5f82fb7c76b2f6a7e35e140bac30/decorator-4.3.0-py2.py3-none-any.whl
Collecting yamlordereddictloader>=0.4.0 (from -r dev-requirements.txt (line 22))
Collecting pytest-cov>=2.4.0 (from -r dev-requirements.txt (line 31))
  Using cached https://files.pythonhosted.org/packages/30/7d/7f6a78ae44a1248ee28cc777586c18b28a1df903470e5d34a6e25712b8aa/pytest_cov-2.5.1-py2.py3-none-any.whl
Collecting Sphinx>=1.5.1 (from -r dev-requirements.txt (line 34))
  Using cached https://files.pythonhosted.org/packages/24/0f/09869a479c626e860451a8b2cc7ed516abd93681b414e3aa9cddbe75f44b/Sphinx-1.7.3-py2.py3-none-any.whl
Collecting sphinx-git>=10.0.0 (from -r dev-requirements.txt (line 35))
  Using cached https://files.pythonhosted.org/packages/91/92/da61745af8728a50e3868d619b154ae3db983ee181cbf256b0cb25c39d92/sphinx_git-10.1.1-py3-none-any.whl
Collecting GitPython>=2.1.1 (from -r dev-requirements.txt (line 36))
  Using cached https://files.pythonhosted.org/packages/88/9c/b462dddb492204417f88d538b0931e87631f2a98afe89842929f4ed9ca5b/GitPython-2.1.9-py2.py3-none-any.whl
Collecting flake8>=3.2.1 (from -r dev-requirements.txt (line 43))
  Using cached https://files.pythonhosted.org/packages/b9/dc/14e9d94c770b8c4ef584e906c7583e74864786a58d47de101f2767d50c0b/flake8-3.5.0-py2.py3-none-any.whl
Collecting twine>=1.8.1 (from -r dev-requirements.txt (line 46))
  Using cached https://files.pythonhosted.org/packages/65/ae/9cfdff186dd4812c59bad890435538858f13fe43dbe6923e0fb20f0adfc8/twine-1.11.0-py2.py3-none-any.whl
Collecting jupyter>=1.0.0 (from -r dev-requirements.txt (line 49))
  Using cached https://files.pythonhosted.org/packages/83/df/0f5dd132200728a86190397e1ea87cd76244e42d39ec5e88efd25b2abd7e/jupyter-1.0.0-py2.py3-none-any.whl
Collecting six>=1.4 (from unittest2>=1.1.0->-r dev-requirements.txt (line 13))
  Using cached https://files.pythonhosted.org/packages/67/4b/141a581104b1f6397bfa78ac9d43d8ad29a7ca43ea90a2d863fe3056e86a/six-1.11.0-py2.py3-none-any.whl
Collecting argparse (from unittest2>=1.1.0->-r dev-requirements.txt (line 13))
  Using cached https://files.pythonhosted.org/packages/f2/94/3af39d34be01a24a6e65433d19e107099374224905f1e0cc6bbe1fd22a2f/argparse-1.4.0-py2.py3-none-any.whl
Collecting traceback2 (from unittest2>=1.1.0->-r dev-requirements.txt (line 13))
  Using cached https://files.pythonhosted.org/packages/17/0a/6ac05a3723017a967193456a2efa0aa9ac4b51456891af1e2353bb9de21e/traceback2-1.4.0-py2.py3-none-any.whl
Requirement not upgraded as not directly required: setuptools in /home/kschopmeyer/.virtualenvs/pywbem3/lib/python3.4/site-packages (from pytest>=3.0.7->-r dev-requirements.txt (line 15)) (39.0.1)
Collecting pluggy<0.7,>=0.5 (from pytest>=3.0.7->-r dev-requirements.txt (line 15))
  Using cached https://files.pythonhosted.org/packages/ba/65/ded3bc40bbf8d887f262f150fbe1ae6637765b5c9534bd55690ed2c0b0f7/pluggy-0.6.0-py3-none-any.whl
Collecting more-itertools>=4.0.0 (from pytest>=3.0.7->-r dev-requirements.txt (line 15))
  Using cached https://files.pythonhosted.org/packages/7a/46/886917c6a4ce49dd3fff250c01c5abac5390d57992751384fe61befc4877/more_itertools-4.1.0-py3-none-any.whl
Collecting py>=1.5.0 (from pytest>=3.0.7->-r dev-requirements.txt (line 15))
  Using cached https://files.pythonhosted.org/packages/67/a5/f77982214dd4c8fd104b066f249adea2c49e25e8703d284382eb5e9ab35a/py-1.5.3-py2.py3-none-any.whl
Collecting attrs>=17.4.0 (from pytest>=3.0.7->-r dev-requirements.txt (line 15))
  Using cached https://files.pythonhosted.org/packages/b5/60/4e178c1e790fd60f1229a9b3cb2f8bc2f4cc6ff2c8838054c142c70b5adc/attrs-17.4.0-py2.py3-none-any.whl
Collecting idna<2.7,>=2.5 (from requests>=2.12.4->-r dev-requirements.txt (line 20))
  Using cached https://files.pythonhosted.org/packages/27/cc/6dd9a3869f15c2edfab863b992838277279ce92663d334df9ecf5106f5c6/idna-2.6-py2.py3-none-any.whl
Collecting chardet<3.1.0,>=3.0.2 (from requests>=2.12.4->-r dev-requirements.txt (line 20))
  Using cached https://files.pythonhosted.org/packages/bc/a9/01ffebfb562e4274b6487b4bb1ddec7ca55ec7510b22e4c51f14098443b8/chardet-3.0.4-py2.py3-none-any.whl
Collecting certifi>=2017.4.17 (from requests>=2.12.4->-r dev-requirements.txt (line 20))
  Using cached https://files.pythonhosted.org/packages/7c/e6/92ad559b7192d846975fc916b65f667c7b8c3a32bea7372340bfe9a15fa5/certifi-2018.4.16-py2.py3-none-any.whl
Collecting urllib3<1.23,>=1.21.1 (from requests>=2.12.4->-r dev-requirements.txt (line 20))
  Using cached https://files.pythonhosted.org/packages/63/cb/6965947c13a94236f6d4b8223e21beb4d576dc72e8130bd7880f600839b8/urllib3-1.22-py2.py3-none-any.whl
Collecting pyyaml (from yamlordereddictloader>=0.4.0->-r dev-requirements.txt (line 22))
Collecting coverage>=3.7.1 (from pytest-cov>=2.4.0->-r dev-requirements.txt (line 31))
  Using cached https://files.pythonhosted.org/packages/28/9d/a35b5281e60ef3c8f42a29da5f60811cd2739c0a6f5bd5342d4115eede5e/coverage-4.5.1-cp34-cp34m-manylinux1_x86_64.whl
Collecting Pygments>=2.0 (from Sphinx>=1.5.1->-r dev-requirements.txt (line 34))
  Using cached https://files.pythonhosted.org/packages/02/ee/b6e02dc6529e82b75bb06823ff7d005b141037cb1416b10c6f00fc419dca/Pygments-2.2.0-py2.py3-none-any.whl
Collecting alabaster<0.8,>=0.7 (from Sphinx>=1.5.1->-r dev-requirements.txt (line 34))
  Using cached https://files.pythonhosted.org/packages/2e/c3/9b7dcd8548cf2c00531763ba154e524af575e8f36701bacfe5bcadc67440/alabaster-0.7.10-py2.py3-none-any.whl
Collecting babel!=2.0,>=1.3 (from Sphinx>=1.5.1->-r dev-requirements.txt (line 34))
  Using cached https://files.pythonhosted.org/packages/94/03/14e68ad12e771a79cf96792f7158d68a7b3d8c7b2badf39e9ef1f65b57d6/Babel-2.5.3-py2.py3-none-any.whl
Collecting imagesize (from Sphinx>=1.5.1->-r dev-requirements.txt (line 34))
  Using cached https://files.pythonhosted.org/packages/e9/79/31cc1c2e0daf575f8fd2b581e2975e6a6938bd439581f766b79c50479521/imagesize-1.0.0-py2.py3-none-any.whl
Collecting sphinxcontrib-websupport (from Sphinx>=1.5.1->-r dev-requirements.txt (line 34))
  Using cached https://files.pythonhosted.org/packages/56/0f/3ee19ca5e5a1d9751cf4bbeb372d40a46421c4321fe55a4703ba66d0bafb/sphinxcontrib_websupport-1.0.1-py2.py3-none-any.whl
Collecting snowballstemmer>=1.1 (from Sphinx>=1.5.1->-r dev-requirements.txt (line 34))
  Using cached https://files.pythonhosted.org/packages/d4/6c/8a935e2c7b54a37714656d753e4187ee0631988184ed50c0cf6476858566/snowballstemmer-1.2.1-py2.py3-none-any.whl
Collecting Jinja2>=2.3 (from Sphinx>=1.5.1->-r dev-requirements.txt (line 34))
  Using cached https://files.pythonhosted.org/packages/7f/ff/ae64bacdfc95f27a016a7bed8e8686763ba4d277a78ca76f32659220a731/Jinja2-2.10-py2.py3-none-any.whl
Collecting docutils>=0.11 (from Sphinx>=1.5.1->-r dev-requirements.txt (line 34))
  Using cached https://files.pythonhosted.org/packages/36/fa/08e9e6e0e3cbd1d362c3bbee8d01d0aedb2155c4ac112b19ef3cae8eed8d/docutils-0.14-py3-none-any.whl
Collecting packaging (from Sphinx>=1.5.1->-r dev-requirements.txt (line 34))
  Using cached https://files.pythonhosted.org/packages/ad/c2/b500ea05d5f9f361a562f089fc91f77ed3b4783e13a08a3daf82069b1224/packaging-17.1-py2.py3-none-any.whl
Collecting typing; python_version < "3.5" (from Sphinx>=1.5.1->-r dev-requirements.txt (line 34))
  Using cached https://files.pythonhosted.org/packages/05/2b/2b05bf1d5a9dd450447c9a5df3e118a465e5d3cb12b73b7220a5064a403f/typing-3.6.4-py3-none-any.whl
Collecting gitdb2>=2.0.0 (from GitPython>=2.1.1->-r dev-requirements.txt (line 36))
  Using cached https://files.pythonhosted.org/packages/e0/95/c772c13b7c5740ec1a0924250e6defbf5dfdaee76a50d1c47f9c51f1cabb/gitdb2-2.0.3-py2.py3-none-any.whl
Collecting pyflakes<1.7.0,>=1.5.0 (from flake8>=3.2.1->-r dev-requirements.txt (line 43))
  Using cached https://files.pythonhosted.org/packages/d7/40/733bcc64da3161ae4122c11e88269f276358ca29335468005cb0ee538665/pyflakes-1.6.0-py2.py3-none-any.whl
Collecting pycodestyle<2.4.0,>=2.0.0 (from flake8>=3.2.1->-r dev-requirements.txt (line 43))
  Using cached https://files.pythonhosted.org/packages/e4/81/78fe51eb4038d1388b7217dd63770b0f428370207125047312886c923b26/pycodestyle-2.3.1-py2.py3-none-any.whl
Collecting mccabe<0.7.0,>=0.6.0 (from flake8>=3.2.1->-r dev-requirements.txt (line 43))
  Using cached https://files.pythonhosted.org/packages/87/89/479dc97e18549e21354893e4ee4ef36db1d237534982482c3681ee6e7b57/mccabe-0.6.1-py2.py3-none-any.whl
Collecting pkginfo>=1.4.2 (from twine>=1.8.1->-r dev-requirements.txt (line 46))
  Using cached https://files.pythonhosted.org/packages/a3/fe/f32a48d48f40a7209be9825fba2566cab92364787cf37de2e08300dd6ce7/pkginfo-1.4.2-py2.py3-none-any.whl
Collecting requests-toolbelt>=0.8.0 (from twine>=1.8.1->-r dev-requirements.txt (line 46))
  Using cached https://files.pythonhosted.org/packages/97/8a/d710f792d6f6ecc089c5e55b66e66c3f2f35516a1ede5a8f54c13350ffb0/requests_toolbelt-0.8.0-py2.py3-none-any.whl
Collecting tqdm>=4.14 (from twine>=1.8.1->-r dev-requirements.txt (line 46))
  Using cached https://files.pythonhosted.org/packages/78/bc/de067ab2d700b91717dc5459d86a1877e2df31abfb90ab01a5a5a5ce30b4/tqdm-4.23.0-py2.py3-none-any.whl
Collecting ipykernel (from jupyter>=1.0.0->-r dev-requirements.txt (line 49))
  Using cached https://files.pythonhosted.org/packages/ab/3f/cd624c835aa3336a9110d0a99e15070f343b881b7d651ab1375ef226a3ac/ipykernel-4.8.2-py3-none-any.whl
Collecting notebook (from jupyter>=1.0.0->-r dev-requirements.txt (line 49))
  Using cached https://files.pythonhosted.org/packages/96/1f/e113875e6b8c8ee0e8c5748bf6fc942a109874adc9ca25248e95cf069803/notebook-5.4.1-py2.py3-none-any.whl
Collecting nbconvert (from jupyter>=1.0.0->-r dev-requirements.txt (line 49))
  Using cached https://files.pythonhosted.org/packages/39/ea/280d6c0d92f8e3ca15fd798bbcc2ea141489f9539de7133d8fe10ea4b049/nbconvert-5.3.1-py2.py3-none-any.whl
Collecting ipywidgets (from jupyter>=1.0.0->-r dev-requirements.txt (line 49))
  Using cached https://files.pythonhosted.org/packages/7d/24/fabc09ad81c6071159a4d12d5bfbddcbea69bd9e3b16c3250ef300c0285f/ipywidgets-7.2.1-py2.py3-none-any.whl
Collecting jupyter-console (from jupyter>=1.0.0->-r dev-requirements.txt (line 49))
  Using cached https://files.pythonhosted.org/packages/77/82/6469cd7fccf7958cbe5dce2e623f1e3c5e27f1bb1ad36d90519bc2d5d370/jupyter_console-5.2.0-py2.py3-none-any.whl
Collecting qtconsole (from jupyter>=1.0.0->-r dev-requirements.txt (line 49))
  Using cached https://files.pythonhosted.org/packages/90/ff/047e0dca2627b162866920e7aa93f04523c0ae81e5c67060eec85701992d/qtconsole-4.3.1-py2.py3-none-any.whl
Collecting linecache2 (from traceback2->unittest2>=1.1.0->-r dev-requirements.txt (line 13))
  Using cached https://files.pythonhosted.org/packages/c7/a3/c5da2a44c85bfbb6eebcfc1dde24933f8704441b98fdde6528f4831757a6/linecache2-1.0.0-py2.py3-none-any.whl
Collecting pytz>=0a (from babel!=2.0,>=1.3->Sphinx>=1.5.1->-r dev-requirements.txt (line 34))
  Using cached https://files.pythonhosted.org/packages/dc/83/15f7833b70d3e067ca91467ca245bae0f6fe56ddc7451aa0dc5606b120f2/pytz-2018.4-py2.py3-none-any.whl
Collecting MarkupSafe>=0.23 (from Jinja2>=2.3->Sphinx>=1.5.1->-r dev-requirements.txt (line 34))
Collecting pyparsing>=2.0.2 (from packaging->Sphinx>=1.5.1->-r dev-requirements.txt (line 34))
  Using cached https://files.pythonhosted.org/packages/6a/8a/718fd7d3458f9fab8e67186b00abdd345b639976bc7fb3ae722e1b026a50/pyparsing-2.2.0-py2.py3-none-any.whl
Collecting smmap2>=2.0.0 (from gitdb2>=2.0.0->GitPython>=2.1.1->-r dev-requirements.txt (line 36))
  Using cached https://files.pythonhosted.org/packages/e3/59/4e22f692e65f5f9271252a8e63f04ce4ad561d4e06192478ee48dfac9611/smmap2-2.0.3-py2.py3-none-any.whl
Collecting traitlets>=4.1.0 (from ipykernel->jupyter>=1.0.0->-r dev-requirements.txt (line 49))
  Using cached https://files.pythonhosted.org/packages/93/d6/abcb22de61d78e2fc3959c964628a5771e47e7cc60d53e9342e21ed6cc9a/traitlets-4.3.2-py2.py3-none-any.whl
Collecting ipython>=4.0.0 (from ipykernel->jupyter>=1.0.0->-r dev-requirements.txt (line 49))
  Using cached https://files.pythonhosted.org/packages/3a/eb/38a4491ebdbb0d5be6f72abc177a7a49f82541bbebbe5fbfe64b041a782b/ipython-6.3.1-py3-none-any.whl
Collecting jupyter-client (from ipykernel->jupyter>=1.0.0->-r dev-requirements.txt (line 49))
  Using cached https://files.pythonhosted.org/packages/94/dd/fe6c4d683b09eb05342bd2816b7779663f71762b4fa9c2d5203d35d17354/jupyter_client-5.2.3-py2.py3-none-any.whl
Collecting tornado>=4.0 (from ipykernel->jupyter>=1.0.0->-r dev-requirements.txt (line 49))
Collecting terminado>=0.8.1 (from notebook->jupyter>=1.0.0->-r dev-requirements.txt (line 49))
  Using cached https://files.pythonhosted.org/packages/2e/20/a26211a24425923d46e1213b376a6ee60dc30bcdf1b0c345e2c3769deb1c/terminado-0.8.1-py2.py3-none-any.whl
Collecting Send2Trash (from notebook->jupyter>=1.0.0->-r dev-requirements.txt (line 49))
  Using cached https://files.pythonhosted.org/packages/49/46/c3dc27481d1cc57b9385aff41c474ceb7714f7935b1247194adae45db714/Send2Trash-1.5.0-py3-none-any.whl
Collecting jupyter-core>=4.4.0 (from notebook->jupyter>=1.0.0->-r dev-requirements.txt (line 49))
  Using cached https://files.pythonhosted.org/packages/1d/44/065d2d7bae7bebc06f1dd70d23c36da8c50c0f08b4236716743d706762a8/jupyter_core-4.4.0-py2.py3-none-any.whl
Collecting nbformat (from notebook->jupyter>=1.0.0->-r dev-requirements.txt (line 49))
  Using cached https://files.pythonhosted.org/packages/da/27/9a654d2b6cc1eaa517d1c5a4405166c7f6d72f04f6e7eea41855fe808a46/nbformat-4.4.0-py2.py3-none-any.whl
Collecting ipython-genutils (from notebook->jupyter>=1.0.0->-r dev-requirements.txt (line 49))
  Using cached https://files.pythonhosted.org/packages/fa/bc/9bd3b5c2b4774d5f33b2d544f1460be9df7df2fe42f352135381c347c69a/ipython_genutils-0.2.0-py2.py3-none-any.whl
Collecting mistune>=0.7.4 (from nbconvert->jupyter>=1.0.0->-r dev-requirements.txt (line 49))
  Using cached https://files.pythonhosted.org/packages/c8/8c/87f4d359438ba0321a2ae91936030110bfcc62fef752656321a72b8c1af9/mistune-0.8.3-py2.py3-none-any.whl
Collecting pandocfilters>=1.4.1 (from nbconvert->jupyter>=1.0.0->-r dev-requirements.txt (line 49))
Collecting bleach (from nbconvert->jupyter>=1.0.0->-r dev-requirements.txt (line 49))
  Using cached https://files.pythonhosted.org/packages/30/b6/a8cffbb9ab4b62b557c22703163735210e9cd857d533740c64e1467d228e/bleach-2.1.3-py2.py3-none-any.whl
Collecting entrypoints>=0.2.2 (from nbconvert->jupyter>=1.0.0->-r dev-requirements.txt (line 49))
  Using cached https://files.pythonhosted.org/packages/cc/8b/4eefa9b47f1910b3d2081da67726b066e379b04ca897acfe9f92bac56147/entrypoints-0.2.3-py2.py3-none-any.whl
Collecting testpath (from nbconvert->jupyter>=1.0.0->-r dev-requirements.txt (line 49))
  Using cached https://files.pythonhosted.org/packages/15/19/d7bc380c479a184e4a5a9ce516e4e2a773165f89b445f7cdced83d94de25/testpath-0.3.1-py2.py3-none-any.whl
Collecting widgetsnbextension~=3.2.0 (from ipywidgets->jupyter>=1.0.0->-r dev-requirements.txt (line 49))
  Using cached https://files.pythonhosted.org/packages/ff/fa/64acc09fc845a6b2dc0724d6f3f81e829b778ed5e9a7559567b4f19a3f4b/widgetsnbextension-3.2.1-py2.py3-none-any.whl
Collecting prompt-toolkit<2.0.0,>=1.0.0 (from jupyter-console->jupyter>=1.0.0->-r dev-requirements.txt (line 49))
  Using cached https://files.pythonhosted.org/packages/04/d1/c6616dd03701e7e2073f06d5c3b41b012256e42b72561f16a7bd86dd7b43/prompt_toolkit-1.0.15-py3-none-any.whl
Collecting backcall (from ipython>=4.0.0->ipykernel->jupyter>=1.0.0->-r dev-requirements.txt (line 49))
Collecting pexpect; sys_platform != "win32" (from ipython>=4.0.0->ipykernel->jupyter>=1.0.0->-r dev-requirements.txt (line 49))
  Using cached https://files.pythonhosted.org/packages/b9/3c/e51a76fe78a877afd9acd95a51e3b7610c0983b19ef5a5dfa11735a619b8/pexpect-4.5.0-py2.py3-none-any.whl
Collecting jedi>=0.10 (from ipython>=4.0.0->ipykernel->jupyter>=1.0.0->-r dev-requirements.txt (line 49))
  Using cached https://files.pythonhosted.org/packages/e7/42/074192a165622e645ed4aeade63e76e56b3496a044569b3c6cae3a918352/jedi-0.12.0-py2.py3-none-any.whl
Collecting simplegeneric>0.8 (from ipython>=4.0.0->ipykernel->jupyter>=1.0.0->-r dev-requirements.txt (line 49))
Collecting pickleshare (from ipython>=4.0.0->ipykernel->jupyter>=1.0.0->-r dev-requirements.txt (line 49))
  Using cached https://files.pythonhosted.org/packages/9f/17/daa142fc9be6b76f26f24eeeb9a138940671490b91cb5587393f297c8317/pickleshare-0.7.4-py2.py3-none-any.whl
Collecting pyzmq>=13 (from jupyter-client->ipykernel->jupyter>=1.0.0->-r dev-requirements.txt (line 49))
  Using cached https://files.pythonhosted.org/packages/a1/d8/2ee889a0420e507ade1564c2271e972abd16b5122229c934e68d2ecf7c6c/pyzmq-17.0.0-cp34-cp34m-manylinux1_x86_64.whl
Collecting python-dateutil>=2.1 (from jupyter-client->ipykernel->jupyter>=1.0.0->-r dev-requirements.txt (line 49))
  Using cached https://files.pythonhosted.org/packages/0c/57/19f3a65bcf6d5be570ee8c35a5398496e10a0ddcbc95393b2d17f86aaaf8/python_dateutil-2.7.2-py2.py3-none-any.whl
Collecting backports-abc>=0.4 (from tornado>=4.0->ipykernel->jupyter>=1.0.0->-r dev-requirements.txt (line 49))
  Using cached https://files.pythonhosted.org/packages/7d/56/6f3ac1b816d0cd8994e83d0c4e55bc64567532f7dc543378bd87f81cebc7/backports_abc-0.5-py2.py3-none-any.whl
Collecting ptyprocess; os_name != "nt" (from terminado>=0.8.1->notebook->jupyter>=1.0.0->-r dev-requirements.txt (line 49))
  Using cached https://files.pythonhosted.org/packages/ff/4e/fa4a73ccfefe2b37d7b6898329e7dbcd1ac846ba3a3b26b294a78a3eb997/ptyprocess-0.5.2-py2.py3-none-any.whl
Collecting jsonschema!=2.5.0,>=2.4 (from nbformat->notebook->jupyter>=1.0.0->-r dev-requirements.txt (line 49))
  Using cached https://files.pythonhosted.org/packages/77/de/47e35a97b2b05c2fadbec67d44cfcdcd09b8086951b331d82de90d2912da/jsonschema-2.6.0-py2.py3-none-any.whl
Collecting html5lib!=1.0b1,!=1.0b2,!=1.0b3,!=1.0b4,!=1.0b5,!=1.0b6,!=1.0b7,!=1.0b8,>=0.99999999pre (from bleach->nbconvert->jupyter>=1.0.0->-r dev-requirements.txt (line 49))
  Using cached https://files.pythonhosted.org/packages/a5/62/bbd2be0e7943ec8504b517e62bab011b4946e1258842bc159e5dfde15b96/html5lib-1.0.1-py2.py3-none-any.whl
Collecting wcwidth (from prompt-toolkit<2.0.0,>=1.0.0->jupyter-console->jupyter>=1.0.0->-r dev-requirements.txt (line 49))
  Using cached https://files.pythonhosted.org/packages/7e/9f/526a6947247599b084ee5232e4f9190a38f398d7300d866af3ab571a5bfe/wcwidth-0.1.7-py2.py3-none-any.whl
Collecting parso>=0.2.0 (from jedi>=0.10->ipython>=4.0.0->ipykernel->jupyter>=1.0.0->-r dev-requirements.txt (line 49))
  Using cached https://files.pythonhosted.org/packages/63/69/0c383765d192006330bfaae27a2c5d0933fb4760c5aacd933b8b19391693/parso-0.2.0-py2.py3-none-any.whl
Collecting webencodings (from html5lib!=1.0b1,!=1.0b2,!=1.0b3,!=1.0b4,!=1.0b5,!=1.0b6,!=1.0b7,!=1.0b8,>=0.99999999pre->bleach->nbconvert->jupyter>=1.0.0->-r dev-requirements.txt (line 49))
  Using cached https://files.pythonhosted.org/packages/f4/24/2a3e3df732393fed8b3ebf2ec078f05546de641fe1b667ee316ec1dcf3b7/webencodings-0.5.1-py2.py3-none-any.whl
Installing collected packages: six, argparse, linecache2, traceback2, unittest2, pluggy, more-itertools, py, attrs, pytest, testfixtures, httpretty, lxml, idna, chardet, certifi, urllib3, requests, decorator, pyyaml, yamlordereddictloader, coverage, pytest-cov, Pygments, alabaster, pytz, babel, imagesize, sphinxcontrib-websupport, snowballstemmer, MarkupSafe, Jinja2, docutils, pyparsing, packaging, typing, Sphinx, smmap2, gitdb2, GitPython, sphinx-git, pyflakes, pycodestyle, mccabe, flake8, pkginfo, requests-toolbelt, tqdm, twine, ipython-genutils, traitlets, backcall, wcwidth, prompt-toolkit, ptyprocess, pexpect, parso, jedi, simplegeneric, pickleshare, ipython, pyzmq, jupyter-core, backports-abc, tornado, python-dateutil, jupyter-client, ipykernel, mistune, pandocfilters, webencodings, html5lib, bleach, jsonschema, nbformat, entrypoints, testpath, nbconvert, terminado, Send2Trash, notebook, widgetsnbextension, ipywidgets, jupyter-console, qtconsole, jupyter
Successfully installed GitPython-2.1.9 Jinja2-2.10 MarkupSafe-1.0 Pygments-2.2.0 Send2Trash-1.5.0 Sphinx-1.7.3 alabaster-0.7.10 argparse-1.4.0 attrs-17.4.0 babel-2.5.3 backcall-0.1.0 backports-abc-0.5 bleach-2.1.3 certifi-2018.4.16 chardet-3.0.4 coverage-4.5.1 decorator-4.3.0 docutils-0.14 entrypoints-0.2.3 flake8-3.5.0 gitdb2-2.0.3 html5lib-1.0.1 httpretty-0.8.14 idna-2.6 imagesize-1.0.0 ipykernel-4.8.2 ipython-6.3.1 ipython-genutils-0.2.0 ipywidgets-7.2.1 jedi-0.12.0 jsonschema-2.6.0 jupyter-1.0.0 jupyter-client-5.2.3 jupyter-console-5.2.0 jupyter-core-4.4.0 linecache2-1.0.0 lxml-4.2.1 mccabe-0.6.1 mistune-0.8.3 more-itertools-4.1.0 nbconvert-5.3.1 nbformat-4.4.0 notebook-5.4.1 packaging-17.1 pandocfilters-1.4.2 parso-0.2.0 pexpect-4.5.0 pickleshare-0.7.4 pkginfo-1.4.2 pluggy-0.6.0 prompt-toolkit-1.0.15 ptyprocess-0.5.2 py-1.5.3 pycodestyle-2.3.1 pyflakes-1.6.0 pyparsing-2.2.0 pytest-3.5.0 pytest-cov-2.5.1 python-dateutil-2.7.2 pytz-2018.4 pyyaml-3.12 pyzmq-17.0.0 qtconsole-4.3.1 requests-2.18.4 requests-toolbelt-0.8.0 simplegeneric-0.8.1 six-1.11.0 smmap2-2.0.3 snowballstemmer-1.2.1 sphinx-git-10.1.1 sphinxcontrib-websupport-1.0.1 terminado-0.8.1 testfixtures-5.4.0 testpath-0.3.1 tornado-5.0.2 tqdm-4.23.0 traceback2-1.4.0 traitlets-4.3.2 twine-1.11.0 typing-3.6.4 unittest2-1.1.0 urllib3-1.22 wcwidth-0.1.7 webencodings-0.5.1 widgetsnbextension-3.2.1 yamlordereddictloader-0.4.0
makefile: Done installing Python development requirements
makefile: Target develop done.
(pywbem3) kschopmeyer@sheldon:~/pywbem/githubpywbem/pywbem(ks/#1187-simple-schema-install)$ 

make build

a-install)$ make build 2>&1 | tee tmp.txt
makefile: Creating the manifest input file
echo "# file GENERATED by makefile, do NOT edit" >MANIFEST.in
echo "LICENSE.txt README.rst INSTALL.md *.py pywbem/*.py pywbem_mock/*.py " |xargs -n 1 echo include >>MANIFEST.in
makefile: Done creating the manifest input file: MANIFEST.in
makefile: Creating the LEX/YACC table modules
rm -f pywbem/mofparsetab.py* pywbem/moflextab.py*
python -c "from pywbem import mof_compiler; mof_compiler._build(verbose=True)"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/kschopmeyer/pywbem/githubpywbem/pywbem/pywbem/__init__.py", line 45, in <module>
    from .mof_compiler import *  # noqa: F403,F401
  File "/home/kschopmeyer/pywbem/githubpywbem/pywbem/pywbem/mof_compiler.py", line 89, in <module>
    from ply import yacc, lex
ImportError: No module named 'ply'
make: *** [pywbem/moflextab.py] Error 1
(pywbem3) kschopmeyer@sheldon:~/pywbem/githubpywbem/pywbem(ks/#1187-simple-schema-install)$

```

--- a/makefile
+++ b/makefile
@@ -199,7 +199,7 @@ help:
 	@echo 'Makefile for $(package_name) project'
 	@echo 'Package version will be: $(package_version)'
 	@echo 'Uses the currently active Python environment: Python $(python_version)'
-	@echo "Valid targets are (they do just what is stated, i.e. no automatic prereq targets):"
+	@echo "Valid targets are:"
 	@echo "  install    - Install pywbem and its Python installation and runtime prereqs (includes install_os once after clobber)"
 	@echo "  develop    - Install Python development prereqs (includes develop_os once after clobber)"
 	@echo "  build      - Build the distribution archive files in: $(dist_dir)"
@@ -224,14 +224,13 @@ help:
 .PHONY: _check_version
 _check_version:
 ifeq (,$(package_version))
-	@echo 'Error: Package version could not be determined (requires pbr; run "make develop")'
+	@echo 'Error: Package version could not be determined (requires pbr; run "make install")'
 	@false
 else
 	@true
 endif
 
-.PHONY: _install_basic
-_install_basic:
+install_basic.done:
 	@echo "makefile: Installing/upgrading basic Python packages (pip, etc., with PACKAGE_LEVEL=$(PACKAGE_LEVEL))"
 ifeq ($(python_mn_version),26)
 	$(PIP_CMD) install importlib
@@ -246,14 +245,12 @@ else
 	$(PYTHON_CMD) -m pip install $(pip_level_opts) pip setuptools wheel
 endif
 	$(PIP_CMD) install $(pip_level_opts) pbr
+	touch install_basic.done
 	@echo "makefile: Done installing/upgrading basic Python packages"
 
 .PHONY: install_os
-install_os: _fresh_install_os install_os.done
+install_os: install_os.done
 	@echo "makefile: Target $@ done."
-
-_fresh_install_os:
-	rm -f install_os.done
 
 install_os.done: pywbem_os_setup.sh
 	@echo "makefile: Installing OS-level installation and runtime requirements"
@@ -269,8 +266,7 @@ _show_bitsize:
 	$(PYTHON_CMD) -c "import platform; print(int(platform.architecture()[0].rstrip('bit')))"
 	@echo "makefile: Done determining bit size of Python executable"
 
-.PHONY: _install_pywbem
-_install_pywbem: requirements.txt win32-requirements.txt win64-requirements.txt setup.py setup.cfg
+install_pywbem.done: requirements.txt win32-requirements.txt win64-requirements.txt setup.py setup.cfg
 	@echo "makefile: Installing pywbem (editable) and its Python runtime prerequisites (with PACKAGE_LEVEL=$(PACKAGE_LEVEL))"
 	rm -Rf build $(package_name).egg-info .eggs
 	$(PIP_CMD) install $(pip_level_opts) -r requirements.txt
@@ -281,20 +277,21 @@ ifeq ($(PYTHON_ARCH),64)
 	$(PIP_CMD) install $(pip_level_opts) -r win64-requirements.txt
 endif
 	$(PIP_CMD) install $(pip_level_opts) -e .
+	touch install_pywbem.done
 	@echo "makefile: Done installing pywbem and its Python runtime prerequisites"
 
 .PHONY: install
-install: makefile install_os.done _install_basic _show_bitsize _install_pywbem $(moftab_files)
+install: install.done
+	@echo "makefile: Target $@ done."
+
+install.done: makefile install_os.done install_basic.done install_pywbem.done
 	$(PYTHON_CMD) -c "import $(package_name); print('ok, version=%r'%$(package_name).__version__)"
 	$(PYTHON_CMD) -c "import $(mock_package_name); print('ok')"
-	@echo "makefile: Target $@ done."
+	touch install.done
 
 .PHONY: develop_os
-develop_os: _fresh_develop_os develop_os.done
+develop_os: develop_os.done
 	@echo "makefile: Target $@ done."
-
-_fresh_develop_os:
-	rm -f develop_os.done
 
 develop_os.done: pywbem_os_setup.sh
 	@echo "makefile: Installing OS-level development requirements"
@@ -303,11 +300,14 @@ develop_os.done: pywbem_os_setup.sh
 	@echo "makefile: Done installing OS-level development requirements"
 
 .PHONY: develop
-develop: develop_os.done _install_basic dev-requirements.txt
+develop: develop.done
+	@echo "makefile: Target $@ done."
+
+develop.done: install.done develop_os.done install_basic.done dev-requirements.txt
 	@echo "makefile: Installing Python development requirements (with PACKAGE_LEVEL=$(PACKAGE_LEVEL))"
 	$(PIP_CMD) install $(pip_level_opts) -r dev-requirements.txt
+	touch develop.done
 	@echo "makefile: Done installing Python development requirements"
-	@echo "makefile: Target $@ done."
 
 .PHONY: build
 build: _check_version $(bdist_file) $(sdist_file)
@@ -457,7 +457,7 @@ $(bdist_file) $(sdist_file): setup.py MANIFEST.in $(dist_dependent_files) $(moft
 	@echo "makefile: Done creating the distribution archive files: $@"
 
 # Note: The mof*tab files need to be removed in order to rebuild them (make rules vs. ply rules)
-$(moftab_files): $(moftab_dependent_files) build_moftab.py
+$(moftab_files): install.done $(moftab_dependent_files) build_moftab.py
 	@echo "makefile: Creating the LEX/YACC table modules"
 	rm -f $(package_name)/mofparsetab.py* $(package_name)/moflextab.py*
 	$(PYTHON_CMD) -c "from pywbem import mof_compiler; mof_compiler._build(verbose=True)"
@@ -498,14 +498,14 @@ else
 	@echo "makefile: Done running Flake8; Log file: $@"
 endif
 
-$(test_log_file): makefile $(package_name)/*.py testsuite/*.py testsuite/testclient/*.py coveragerc
+$(test_log_file): makefile $(package_name)/*.py $(mock_package_name)/*.py testsuite/*.py testsuite/test_uprint.* testsuite/testclient/*.py testsuite/testclient/*.yaml coveragerc
 	@echo "makefile: Running tests"
+	rm -f $(test_log_file)
 ifeq ($(PLATFORM),Windows)
 	testsuite\test_uprint.bat
 else
 	testsuite/test_uprint.sh
 endif
-	rm -f $(test_log_file)
 	bash -c "set -o pipefail; PYTHONWARNINGS=default py.test --cov $(package_name) --cov $(mock_package_name) $(coverage_report) --cov-config coveragerc --ignore=attic --ignore=releases -s 2>&1 |tee $(test_tmp_file)"
 	mv -f $(test_tmp_file) $(test_log_file)
 	@echo "makefile: Done running tests; Log file: $@"
@@ -519,4 +519,3 @@ $(doc_conf_dir)/mof_compiler.help.txt: mof_compiler $(package_name)/mof_compiler
 	@echo "makefile: Creating mof_compiler script help message file"
 	./mof_compiler --help >$@
 	@echo "makefile: Done creating mof_compiler script help message file: $@"
-

